cypress/e2e/tests/migration/applicationinventory/applications/manage_columns.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    openManageColumns,
+    restoreColumnsToDefault,
+    validateTextPresence,
+    validateCheckBoxIsDisabled,
+    clickByText,
+} from "../../../../../utils/utils";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+import { button, save, SEC } from "../../../../types/constants";
+import { manageColumnsModal } from "../../../../views/applicationinventory.view";
+import { getCheckboxSelector } from "../../../../views/common.view";
+
+const applicationInventoryTableColumns = [
+    "Name",
+    "Business Service",
+    "Assessment",
+    "Review",
+    "Analysis",
+    "Tags",
+    "Effort",
+];
+const columnsToShuffleAndTest = [
+    "Business Service",
+    "Assessment",
+    "Review",
+    "Analysis",
+    "Tags",
+    "Effort",
+];
```

Comment:
You can use the spread operator here to avoid repeating yourself

        const columnsToShuffleAndTest = [
            "Business Service",
            "Assessment",
            "Review",
            "Analysis",
            "Tags",
            "Effort",
        ];
        const applicationInventoryTableColumns = [
            "Name",
            ...columnsToShuffleAndTest
        ];

Reply:
Done!